src/test/ui/inference/need_type_info/do-not-suggest-generic-arguments-for-turbofish.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0282]: type annotations needed
+  --> $DIR/do-not-suggest-generic-arguments-for-turbofish.rs:8:5
+   |
+LL |     OhNo::C::<u32, _>;
+   |     ^^^^^^^^^^^^^^^^^ cannot infer type of the type parameter `U` declared on the enum `OhNo`
```

Comment:
I think this suggestion should point at the specific type parameter. I will work on it later.